app/src/Commands/Pack/PackCreateCommand.php

```diff
@@ -0,0 +1,712 @@
+
+    protected Docker $docker;
+    protected RepoManager $repo_manager;
+    protected Git $git;
+    protected Posix $posix;
+    protected Filesystem $filesystem;
+    protected Linux $linux;
+    protected ProjectConfig $project_config;
+    protected Writer $writer;
+
+    public function __construct(
+        Docker $docker,
+        RepoManager $repo_manager,
+        Git $git,
+        Posix $posix,
+        Filesystem $filesystem,
+        Linux $linux,
+        ProjectConfig $project_config,
+        Writer $writer
+    ) {
+        parent::__construct();
+
+        $this->docker = $docker;
+        $this->repo_manager = $repo_manager;
+        $this->git = $git;
+        $this->posix = $posix;
+        $this->filesystem = $filesystem;
+        $this->linux = $linux;
+        $this->project_config = $project_config;
+        $this->writer = $writer;
+    }
+
+    public function configure() : void
+    {
+        $this
+            ->setAliases(["pack-create"])
+            ->addOption("name", "e", InputOption::VALUE_OPTIONAL, "Sets the name of the instance")
+            ->addOption("repo", "r", InputOption::VALUE_OPTIONAL, "Sets the repository to use")
+            ->addOption("use-global-repo", "u", InputOption::VALUE_NONE, "Determines if the repo is global one or not")
+            ->addOption("branch", "b", InputOption::VALUE_OPTIONAL, "Sets the branch to use")
+            ->addOption("phpversion", "p", InputOption::VALUE_OPTIONAL, "Sets the php version to use")
+            ->addOption("target", "t", InputOption::VALUE_OPTIONAL, "Sets the target destination for the instance. If the folder does not exist, it will be created. Will be ignored while creating global instances")
+            ->addOption("xdebug", "x", InputOption::VALUE_NONE, "Determines if xdebug should be installed or not")
+            ->addOption("global", "g", InputOption::VALUE_NONE, "Determines if an instance is global or not")
+            ->addOption("skip-readme", "s", InputOption::VALUE_NONE, "Doesn't create the README.md file")
+            ->setHidden(true)
+        ;
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output) : int
+    {
+        $options = $this->gatherOptionData($input, $output);
+
+        $instance_path = $options["target"] . "/" . $options["name"];
+        $suffix = $options["global"] ? "global" : "local";
+        $instance_name = $options["name"] . "_" . $suffix;
+        $instance_salt_name = $options["name"] . "." . $suffix;
+        $user_name = $this->posix->getCurrentUserName();
+        $home_dir = $this->posix->getHomeDirectory($this->posix->getUserId());
+
+        if ($this->filesystem->exists($instance_path)) {
+            $this->writer->error(
+                $output,
+                $options["name"] . " already exists!",
+                "See <fg=gray>doil instances:create --help</> for more information."
```

Comment:
Hier muss es auch `pack:create` heißen, oder?

Reply:
Da dies ein verstecktes Kommando ist, hat der User keinen Zugriff auf pack:create. In dem Fall muss es bei instances:create bleiben.